SwiftyTimer.podspec

```diff
@@ -1,17 +1,20 @@
 Pod::Spec.new do |s|
   s.name = 'SwiftyTimer'
-  s.version = '2.1.0'
+  s.version = '2.1.1'
   s.license = 'MIT'
   s.summary = 'Swifty API for NSTimer'
   s.homepage = 'https://github.com/radex/SwiftyTimer'
   s.authors = { 'Radek Pietruszewski' => 'this.is@radex.io' }
   s.source = { git: 'https://github.com/radex/SwiftyTimer.git', tag: s.version }
-  s.swift_version = '4.2'
+
+  s.swift_version = '5.0'
```

Comment:
This line should not be required as `swift_versions` is defined later on and overwrites it.

Reply:
At the time when I added that, swift_versions was new so newer CocoaPods was using it while older ones ignored it. I agree, this can now be updated to just one setting, I believe everyone has moved on.